### ISSUE TITLE
Duotone effect also applied to the Image block border color

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#52142]
I expected that the duotone filter would be applied to the image, not the image's border. 

### To reproduce: 

1. Add an image block. 
2. Add a white border to it.
3. Add a duotone filter.
4. See effect also applied to the image block border. 

### Visual 

I've added an image with a white border applied:

https://github.com/WordPress/gutenberg/assets/1813435/7c645761-ec5d-4d7b-80d3-ecbf6baf67fb

And here's the Cover block performing as expected: 

https://github.com/WordPress/gutenberg/assets/1813435/cc02779b-4973-4df2-a14c-d560cd1405b5

**Comment by justintadlock** [comment]
Not sure if this is strictly a bug because the CSS `filter` property  is working correctly.

_Thinking out loud..._ The usual workaround for this in is to apply the border to a wrapping element, like a `<div>`. That would mean adding extra HTML to the block markup.

Applying the border to the existing `<figure>` would be problematic since it also wraps `<figcaption>`. There might be some possibilities with pseudo-elements, but we'd run the risk of breaking block styles in themes.

**Comment by richtabor** [comment]
> Not sure if this is strictly a bug because the CSS filter property is working correctly.
> Thinking out loud... The usual workaround for this in is to apply the border to a wrapping element, like a div.

It's not doing it on the Cover block though. I'd expect the same result for both. 😅

**Comment by justintadlock** [comment]
> It's not doing it on the Cover block though. I'd expect the same result for both. 😅

Agreed. It looks like the big difference between the two is that the border on the Cover is applied to the wrapper, but the border on the Image is applied to the `<img>`. 

The best course of action might be to introduce a wrapping `<div>` for the `<img>` element for the Image block and apply the border props to it. That's the path I'm seeing with the least disruption. Just with some quick testing, it could make for some minor compatibility breaks (but not insurmountable) for some custom block styles.

![image-block-inner-wrapper](https://github.com/WordPress/gutenberg/assets/1816309/5b49831c-e13f-445e-a00c-252ddce2df92)

**Comment by carolinan** [comment]
I am also not sure if this is a bug or a feature. I kind of like it but I can see why it would not be wanted at all times.

One more inconsistency: When the block has a duotone filter and a border, and you select the "crop" image editing tool from the block toolbar, the border no longer has the duotone applied.

**Comment by richtabor** [comment]
In my opinion, duotone should apply to the image, not the other effects added to the image. Having the border visually be a different color than what's selected in the control is not an ideal experience.
